README.md

```diff
@@ -6,7 +6,7 @@ Tags: Redis,Cache,StackExchange.Redis,Microsoft,Azure
 ---
 
 # Microsoft.Azure.StackExchangeRedis Extension
-The Microsoft.Azure.StackExchangeRedis package is an extension for the StackExchange.Redis client library that enables using Azure Active Directory to authenticate connections from a Redis client application to an Azure Cache for Redis resource. This extension acquires an authentication token for a manged identity or service principal and configures a StackExchange.Redis connection to use the token for authentication. It also maintains the token, proactively refreshing it and re-authenticating the connection to maintain uninterrupted communication with the cache over multiple days.
+The Microsoft.Azure.StackExchangeRedis package is an extension for the StackExchange.Redis client library that enables using Azure Active Directory to authenticate connections from a Redis client application to an Azure Cache for Redis resource. This extension acquires an authentication token for a [Managed Identity](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) or [Service Principal](https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals) and configures a StackExchange.Redis connection to use the token for authentication. It also maintains the token, proactively refreshing it and re-authenticating the connection to maintain uninterrupted communication with the cache over multiple days.
```

Comment:
Could you remove 'en-us/' from these URLs? That will allow the docs to display in the user's preferred language.

Reply:
Done, nice to learn that trick.